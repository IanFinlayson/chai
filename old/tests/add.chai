def main():
    var some_v9ar = 999
    print(some_v9ar + 1)
    print(77 + some_v9ar + some_v9ar + some_v9ar + 2)
    print("hey" + " " + "there")
    print([1, 2] + [] + [3, 4] + [5])

