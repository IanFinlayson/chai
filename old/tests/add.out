1000
3076
hey there
[1, 2, 3, 4, 5]
